app/helpers/auth_helper.rb

```diff
@@ -0,0 +1,2 @@
+module AuthHelper
```

Comment:
We might want to get rid of this for now if we're not using it, same with the ones below